posthog/rate_limit.py

```diff
@@ -147,13 +149,23 @@ def team_is_allowed_to_bypass_throttle(self, team_id: Optional[int]) -> bool:
         return team_id is not None and str(team_id) in allow_list
 
 
-class DecideRateThrottle(SimpleRateThrottle):
+class DecideRateThrottle(BaseThrottle):
+    """
+    This is a custom throttle that is used to limit the number of requests to the /decide endpoint.
+    It is different from the TeamRateThrottle in that it does not use the Django cache, but instead
+    uses the Limiter from the `token-bucket` library.
+    This uses the token bucket algorithm to limit the number of requests to the endpoint. It's a lot
+    more performant than DRF's SimpleRateThrottle, which inefficiently uses the Django cache.
 
-    scope = "decide"
+    However, note that this throttle is per process, and not global.
+    """
 
-    def __init__(self, rate: str = "400/m") -> None:
-        self.rate = rate
-        super().__init__()
+    def __init__(self, replenish_rate: float = 5, bucket_capacity=100) -> None:
```

Comment:
```suggestion
    def __init__(self, replenish_rate: float = 5.0, bucket_capacity=100) -> None:
```

Reply:
will address in a follow up - want to get this in so I can run the e2e test on dev without having to push that out further 😓